CMakeLists.txt

```diff
@@ -48,7 +48,8 @@ ADD_LIBRARY(
 		node_lib
 		src/node/node.cc
 		src/log_manager/leader_log_manager.cc
-		src/log_manager/non_leader_log_manager.cc)
+		src/log_manager/non_leader_log_manager.cc
+		src/node/no_op_request.h)
```

Comment:
src/node/no_op_request.h should not be added here

Reply:
hmm, where will be ok?

Reply:
> hmm, where will be ok?

delete it

Reply:
need  no_op_request for leader